package.json

```diff
@@ -1,10 +1,11 @@
 {
   "name": "@mapbox/react-click-to-select",
-  "version": "1.1.1",
+  "version": "2.0.0",
   "description": "a higher-order component that selects elements contained within it on click",
-  "main": "index.js",
+  "main": "dist/index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "prepublish": "npm run build",
```

Comment:
We should make it `prepublishOnly`, so it runs before publish, not after installation.